packages/fetchai/skills/tac_control_contract/behaviours.py

```diff
@@ -113,8 +114,13 @@ def act(self) -> None:
             )
             self.token_ids = [int(token_id) for token_id in token_ids_dictionary.keys()]
             self.context.logger.info("Creating the items.")
+            self.currency_id = self.token_ids[0]
+            self.token_ids = self.token_ids[1:11]
```

Comment:
Splitting the token_ids, it is not ideal but it is the fastest way. Else we have to change the assertion when we are creating the items.

Reply:
See above, then here you assign to `self._*`